boilerplates/airflow/Jenkinsfile

```diff
@@ -0,0 +1,70 @@
+def final projectId = '@project_id@'
+def final componentId = 'airflow-worker'
+def final credentialsId = "${projectId}-cd-cd-user-with-password"
+def sharedLibraryRepository
+def dockerRegistry
+node {
+    sharedLibraryRepository = env.SHARED_LIBRARY_REPOSITORY
+    dockerRegistry = env.DOCKER_REGISTRY
+}
+
+library identifier: 'ods-library@production', retriever: modernSCM(
+        [$class       : 'GitSCMSource',
+         remote       : sharedLibraryRepository,
+         credentialsId: credentialsId])
+
+// See readme of shared library for usage and customization.
+odsPipeline(
+        image: "${dockerRegistry}/cd/jenkins-slave-airflow",
+        projectId: projectId,
+        componentId: componentId,
+        openshiftBuildTimeout: 25,
+        branchToEnvironmentMapping: [
+                'master': 'test',
+                '*'     : 'dev'
+        ]
+) { context ->
+    stageDAGTest(context)
+    stageUnitTest(context)
+    stageBuild(context)
+    stageStartOpenshiftBuild(context)
+    stageDeployToOpenshift(context)
+    stagePublishDAGs(context)
+
+}
+
+def stageDAGTest(def context) {
+    stage('DAG Integrity Tests') {
+        sh './test_dag_integrity.sh'
+    }
+}
+
+def stageUnitTest(def context) {
+    stage('Unit Tests') {
+        sh "pip install -i ${context.nexusHostWithBasicAuth}/repository/pypi-all/simple -r src/requirements.txt --user"
+        sh './test_all.sh'
+        junit 'artifacts/**/*.xml'
```

Comment:
Please change the location or add config, see #299.

Reply:
should be solved now @clemensutschig  @michaelsauter